tests/network/hotplug_sriov.go

```diff
@@ -53,6 +56,23 @@ var _ = Describe(SIG(" SRIOV nic-hotplug", Serial, decorators.SRIOV, func() {
 
 	})
 
+	BeforeEach(func() {
+		virtClient := kubevirt.Client()
+		originalKv := libkubevirt.GetCurrentKv(virtClient)
+		updateStrategy := &v1.KubeVirtWorkloadUpdateStrategy{
+			WorkloadUpdateMethods: []v1.WorkloadUpdateMethod{v1.WorkloadUpdateMethodLiveMigrate},
+		}
+		rolloutStrategy := pointer.P(v1.VMRolloutStrategyLiveUpdate)
```

Comment:
Other tests that set the LiveUpdate rollout strategy add the decorator `VMLiveUpdateRolloutStrategy`. Not sure why honestly, but maybe we want to do the same here.

Reply:
> > Other tests that set the LiveUpdate rollout strategy add the decorator `VMLiveUpdateRolloutStrategy`. Not sure why honestly, but maybe we want to do the same here.
> 
> Is it used anywhere in our CI?

It doesn't appear to be... Found this though: https://github.com/kubevirt/kubevirt/pull/13091/files#r1810163792

Reply:
FYI @fossedihelm

Reply:
If it doesn't do anything, maybe we should consider removing it in a follow-up.